doc/manual/calling-c-and-fortran-code.rst

```diff
@@ -743,14 +736,90 @@ any expression, such as ``&0`` or ``&f(x)``.
 When a scalar value is passed with ``&`` as an argument of type
 ``Ptr{T}``, the value will first be converted to type ``T``.
 
+Some Examples of C Wrappers
+---------------------------
+
+Here is a simple example of a C wrapper that returns a ``Ptr`` type::
+
+    type gsl_permutation
+    end
+
+    # The corresponding C signature is
+    #     gsl_permutation * gsl_permutation_alloc (size_t n);
+    function permutation_alloc(n::Integer)
+        output_ptr = ccall( (:gsl_permutation_alloc, :libgsl),
+            Ptr{gsl_permutation}, (Csize_t,), n )
```

Comment:
Maybe align this line with the parenthesis of the `ccall`? (Also, I don't think spaces inside parentheses are frequent in the Julia codebase.)